hasura/migrations/default/1670234780227_create_table_public_locked_token_distributions/up.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE "public"."locked_token_distributions" ("id" bigserial NOT NULL, "epoch_id" bigint NOT NULL, "gift_amount" numeric NOT NULL, "tx_hash" varchar, "distribution_json" jsonb NOT NULL, "distributed_by" varchar NOT NULL, PRIMARY KEY ("id") , FOREIGN KEY ("epoch_id") REFERENCES "public"."epoches"("id") ON UPDATE cascade ON DELETE cascade);
```

Comment:
What values does `distributed_by` contain?

Reply:
This is the address that kicked off the distribution, we will display it in the locked token distribution table.

Reply:
could we make it into a foreign key linking to `profiles.id`? or do you want the flexibility of being able to add an address that doesn't correspond to a Coordinape user?

Reply:
@levity yes that makes sense, I've updated the database to have a foreign key and added the rule to have it populated by the session variable (similar to how the distributions table is configured).